extensions/amp-mustache/0.1/test/test-amp-mustache.js

```diff
@@ -99,6 +99,24 @@ describes.repeated('amp-mustache 0.1', {
     });
   });
 
+  it('should pass document when sanitizing for AMP format type context', () => {
+    sandbox.spy(sanitizer, 'sanitizeHtml');
+    innerHtmlSetup('value = <a href="{{value}}">abc</a>');
+    template.compileCallback();
+    allowConsoleError(() => {
+      template.render({
+        value: 'https://www.site.org',
+      });
+      expect(sanitizer.sanitizeHtml).to.have.been.calledWith(
+          sinon.match.string,
+          sinon.match(arg => {
+            return !!arg.documentElement;
+          }),
+          false
+      );
+    });
+  });
```

Comment:
Maybe this can be removed now that `doc` is a required param. 

Optional: we could instead add another sanitization test (see test above this one) to make sure that mustache is using purifier/sanitizer correctly.

Reply:
removed.